heartbeat/docs/overview.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[heartbeat-overview]]
+== Overview
+
+//TODO: Add Beta tag, when available, to generate the labels indicating that this is a Beta feature.
+
+Heartbeat is a lightweight daemon that you install on a remote server
+to ping your services periodically and determine whether they are available.
+Unlike {metricbeat}/index.html[Metricbeat], which only tells you if your servers
+are up or down, Heartbeat tells you whether your services are reachable. If
+you run your services using a DNS-based load balancer or proxy, you can configure
```

Comment:
fixed. still says ping but I think that's OK in the context of this particular sentence.